target/classes/css/style.css

```diff
@@ -0,0 +1,27 @@
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    width: 200px;
+    background-color: rgb(219,162,108);
+  }
+  
+  li a {
+    display: block;
+    color: #000;
+    padding: 8px 16px;
+    text-decoration: none;
+  }
+  
+  /* Change the link color on hover */
+  li a:hover {
+    background-color: #555;
+    color: white;
+  }
+
+  #homeheader {
+    display: flex;
+    justify-content: center;
+    background-color: rgb(219,162,108);
+    padding: 15px;
+  }
```

Comment:
When I did my first successful compile, an excluded directory called target had been automatically added afterwards. with files underneath it such as the style.css, any html files, and the one image file included. I also saw these in my own group's project, so I can't imagine the files under the target directory will do any harm.